Sofa/Component/AnimationLoop/src/sofa/component/animationloop/ConstraintAnimationLoop.cpp

```diff
@@ -958,9 +958,13 @@ ConstraintProblem* ConstraintAnimationLoop::getCP()
         return &CP1;
 }
 
-
-int ConstraintAnimationLoopClass = core::RegisterObject ( "Constraint animation loop manager" )
+void registerConstraintAnimationLoop(sofa::core::ObjectFactory* factory)
+{
+    factory->registerObjects(core::ObjectRegistrationData("Constraint animation loop manager")
         .add< ConstraintAnimationLoop >()
-        .addAlias("MasterConstraintSolver");
+        .addAlias("MasterConstraintSolver"));
```

Comment:
```suggestion
```
💥

Reply:
You should have let the `);` in your suggestion, I cannot commit as it is 😗